src/fake_data/provide_hass.ts

```diff
@@ -178,6 +178,7 @@ export const provideHass = (
 
     translationMetadata: translationMetadata as any,
     dockedSidebar: "auto",
+    vibrate: true,
```

Comment:
Let's not put this in `hass` but store this in `coreUserData` instead like we do for advanced mode

Reply:
oh interesting, maybe you are right with this approach, as with this approach it's a per-browser setting and not per-user.

Reply:
Yeah, that was the idea

Reply:
ok, great